src/main/java/org/apache/maven/shared/utils/cli/Commandline.java

```diff
@@ -65,12 +65,14 @@
 public class Commandline
     implements Cloneable
 {
-    private final List<Arg> arguments = new Vector<Arg>();
+    private final List<Arg> arguments = new Vector<>();
```

Comment:
Is Vector needed here?

Reply:
That's old code. I wouldn't touch it in THIS PR.

Reply:
Exactly.
I don't want to mixing changes in one PR.

Probably Vector and below `synchronizedMap` can be replaced for not synchronized collections.

`Commandline` should be one time instance not shared by threads.
But it is another issue.